src/Tizen.Applications.Common/Tizen.Applications/CultureInfoHelper.cs

```diff
@@ -15,32 +15,62 @@
  */
 
 using System;
+using System.IO;
 using System.Runtime.InteropServices;
 
 namespace Tizen.Applications
 {
-    internal static class CultureInfoHelper
+    internal class CultureInfoManager
     {
+        private static CultureInfoManager _inst;
         private const string _pathCultureInfoIni = "/usr/share/dotnet.tizen/framework/i18n/CultureInfo.ini";
+        private static IntPtr _dictionary = IntPtr.Zero;
 
-        public static string GetCultureName(string locale)
+        public static CultureInfoManager GetInst()
+        {
+            return _inst;
+        }
+
+        private CultureInfoManager()
+        {
+            if (File.Exists(_pathCultureInfoIni))
+            {
+                _dictionary = Interop.LibIniParser.Load(_pathCultureInfoIni);
+            }
+        }
+
+        ~CultureInfoManager()
+        {
+            if (_dictionary != IntPtr.Zero)
+            {
+                Interop.LibIniParser.FreeDict(_dictionary);
+            }
```

Comment:
Please do not make using singleton pattern.
In C# language, static class is recommended instead of singleton pattern.

Reply:
I modified this implementation.
Thanks for the good suggestion 👍